include/mbedtls/ecdsa.h

```diff
@@ -241,11 +236,20 @@ int mbedtls_ecdsa_write_signature( mbedtls_ecdsa_context *ctx, mbedtls_md_type_t
  *                  (SECG): SEC1 Elliptic Curve Cryptography</em>, section
  *                  4.1.3, step 5.
  *
- * \return          \c 0 on success,
- *                  or an \c MBEDTLS_ERR_ECP_XXX, \c MBEDTLS_ERR_MPI_XXX or
- *                  \c MBEDTLS_ERR_ASN1_XXX error code on failure.
- *
  * \see             ecp.h
+ *
+ * \deprecated      Superseded by mbedtls_ecdsa_write_signature() in 2.0.0
```

Comment:
Pre-existing, but I don't think 'in 2.0.0' obviously means to the reader, Mbed TLS version 2.0.0.

I suggest we change the text to 'Superseded by mbedtls_ecdsa_write_signature() in Mbed TLS version 2.0 and later'.

Reply:
Agreed and modified